src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/DraftReferralDTO.kt

```diff
@@ -49,7 +49,9 @@ data class DraftReferralDTO(
         serviceUser = ServiceUserDTO.from(referral.serviceUserCRN, referral.serviceUserData),
         serviceProvider = ServiceProviderDTO.from(contract.primeProvider),
         relevantSentenceId = referral.relevantSentenceId,
-        serviceCategoryId = contract.contractType.serviceCategories.elementAt(0).id,
+        // TODO: remove this once cohort referrals changes are complete
+        serviceCategoryId = if (referral.selectedServiceCategories?.isNotEmpty() == true) referral.selectedServiceCategories!!.elementAt(0).id else null,
```

Comment:
not a blocker, i think this could be simplified to just `referral.selectedServiceCategories?.elementAt(0)`